docs/source/api/index.rst

```diff
@@ -1,8 +1,7 @@
 ================
 APIs
 ================
-.. toctree::
-   :maxdepth: 1
-   :caption: APIs
+.. autosummary::
+   :toctree: _autosummary
    
    ansys.dpf.core
```

Comment:
Sadly, this will have to include additional work to auto generate the list of classes within `ansys.dpf.core` as our templates don't generate at the module level recursively. Either this will take the form of a script to generate the APIs, or we'll need to add in an additional template to walk through the API docgen at the module level.